packages/contractkit/src/wallets/aws-hsm-wallet.ts

```diff
@@ -57,11 +67,20 @@ export default class AwsHsmWallet extends RemoteWallet implements Wallet {
       throw new Error('AwsHsmWallet needs to be initialised first')
     }
     const { PublicKey } = await this.kms.getPublicKey({ KeyId: keyId }).promise()
-    return publicKeyFromAsn1(Buffer.from(PublicKey as Uint8Array))
+    const pubKey = publicKeyFromAsn1(Buffer.from(PublicKey as Uint8Array))
+    const pkbuff = bigNumberToBuffer(pubKey, 64)
```

Comment:
64 is hardcoded in a couple places (in and outside of this PR), is something that could be put into a constant?